packages/jest-test-result/src/__tests__/formatTestResults.test.ts

```diff
@@ -32,4 +32,58 @@ describe('formatTestResults', () => {
       assertion.fullName,
     );
   });
+
+  const skippedAssertion = {
+    fullName: 'Pending test',
+    status: 'pending',
+    title: 'is still pending',
+  };
+
+  const skippedResults: AggregatedResult = {
+    testResults: [
+      {
+        numFailingTests: 0,
+        numPassingTests: 0,
+        numPendingTests: 2,
+        numTodoTests: 2,
+        perfStats: {end: 2, runtime: 1, slow: false, start: 1},
+        // @ts-expect-error
```

Comment:
Instead of `@ts-expect-error` it would be better to use casting (see other suggestions) in all tests of this file.

I see that `@ts-expect-error` was already present here and you just followed the pattern. It works, but casting feels somewhat cleaner.

```suggestion
```

Reply:
👍 Applied changes (thank you for the feedback).
The pipeline keeps failing but I don't think it is related to my changes.

Reply:
Perfect. Thanks! Apologies about misleading suggestion before, I just noticed that all `AssertionResult` casts are actually unnecessary. Could you remove them, please?

There is an odd issue with type checks on CI. Hard to debug, because all works smoothly locally. I was trying to find the root of this problem, but no luck so far. Can be ignored for now.

Reply:
Pipelines are a nightmare to debug 😄
When you'll have the time, I have 2 open PRs which should be ready for review (#13639 and #13607).